usaspending_api/awards/models.py

```diff
@@ -258,7 +259,24 @@ class Meta:
         db_table = 'awards'
 
 
-class Transaction(DataSourceTrackedModel):
+class TransactionAgeComparisonMixin:
+
+    def newer_than(self, dct):
+        """Compares age of this instance to a dict
+
+        Compares (last_modified_date or certified_date) of each
```

Comment:
For code readers who aren't super well-versed in Python, it might be worth expanded this description to specifically state that we check last_modified date first and then certified date.